### ISSUE TITLE
Create new content page

### ISSUE THREAD

**Issue by phae** [codeforamerica/ceviche-cms#70]
Probably a button to push to make a duplicate of a folder & index.markdown in the given directory.

**Comment by phae** [comment]
Should also probably set some sort of default template type

**Comment by phae** [comment]
#77 is related to this one

**Comment by phae** [comment]
#70

**Comment by phae** [comment]
#218

**Comment by phae** [comment]
depends on #178
